Musician Module/Controls/SheetButton.cs

```diff
@@ -0,0 +1,219 @@
+﻿using System;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Blish_HUD.Controls;
+using Blish_HUD.Modules.Musician.Notation.Persistance;
+namespace Blish_HUD.Modules.Musician.Controls {
+
+    // TODO: Show "Edit" button when music sheet creator correlates to account name from ApiService. Navigates to composer.
+    public class SheetButton : DetailsButton {
+
+        private const int SHEETBUTTON_WIDTH = 327;
+        private const int SHEETBUTTON_HEIGHT = 100;
+
+        private const int USER_WIDTH = 75;
+        private const int BOTTOMSECTION_HEIGHT = 35;
+
+        public string Artist { get; set; }
+        public string Title { get; set; }
+        public string User { get; set; }
+        private static Texture2D BeatmaniaSprite;
+        private static Texture2D AutoplaySprite;
+        private static Texture2D BackgroundSprite;
+        private static Texture2D DividerSprite;
+        private static Texture2D IconBoxSprite;
+        private Texture2D PreviewSprite;
+        private bool _isPreviewing;
+        public bool IsPreviewing
+        {
+            get => _isPreviewing;
+            set
+            {
+                if (value == _isPreviewing) return;
+                _isPreviewing = value;
+                Invalidate();
+            }
+        }
+        private Texture2D _icon;
+        public Texture2D Icon
+        {
+            get => _icon;
+            set
+            {
+                if (_icon == value) return;
+
+                _icon = value;
+                OnPropertyChanged();
+            }
+        }
+        private DetailsIconSize IconSize;
+        private RawMusicSheet _musicSheet;
+        public RawMusicSheet MusicSheet
+        {
+            get => _musicSheet;
+            set
+            {
+                if (_musicSheet == value) return;
+
+                _musicSheet = value;
+                OnPropertyChanged();
+            }
+        }
+        public SheetButton()
+        {
+
+            BeatmaniaSprite = BeatmaniaSprite ?? Content.GetTexture("beatmania");
+            AutoplaySprite = AutoplaySprite ?? Content.GetTexture("autoplay");
+            BackgroundSprite = BackgroundSprite ?? ContentService.Textures.Pixel;
+            DividerSprite = DividerSprite ?? Content.GetTexture("157218");
+            IconBoxSprite = IconBoxSprite ?? Content.GetTexture("605003");
```

Comment:
All textures should get loaded through the `ContentsManager` that is a property on your main module class (it is used to get textures from your ref that gets bundled with the module that you deploy).  I think `Content` will likely get switched to internal at some point since I forgot about that.  The pixel is still, fine, though.